cmds/tui.c

```diff
@@ -1043,7 +1049,7 @@ static void win_footer_graph(struct tui_window *win,
 		}
 	}
 
-	win_footer(win, buf);
+	win_footer(win, buf, true);
```

Comment:
Instead you can skip calling it for graph.

Reply:
Are you saying that this line has to be removed completely from this pull request? 
Or do I have to move this line to the next commit?

**Cause at next commit, `win_footer` is used to print the `position` at lower right corner.**

If what you meant was not comfortable with adding extra argument `bool is_graph`, I think the same way. 
_I really don't want to add that one for only one purpose which is just comparing which type of window is it._

So what I've thought is, adding type attribute to `tui_window` 
```
#define TUI_SESS_REPORT    1
#define TUI_SESS_INFO      2
#define TUI_SESS_HELP      3
#define TUI_SESS_QUIT      4
#define TUI_SESS_DUMMY_NR  4

struct tui_window {
	const struct tui_window_ops *ops;
	void *top;
	void *curr;
	void *old;
        int type; // <- for storing the type of windows
	int top_index;
	int curr_index;
	int search_count;
};
```

and with this, we don't have to pass extra `is_graph` argument.
What do you think?

Reply:
Actually I don't think it's not that useful for info or session window to display debug message or percent (of position) since it doesn't have many lines usually.

Reply:
In that case, remove debug message from `win_footer' and roll-back debug message to graph and report will solve this and upper issue.

Reply:
Shall I proceed with roll-back to each footer? (Graph, report)